frontend/src/fetchers/ssdi.js

```diff
@@ -5,22 +5,58 @@ import { QA_DASHBOARD_FILTER_CONFIG } from '../pages/QADashboard/constants';
 
 const ssdiUrl = join('/', 'api', 'ssdi');
 
+const urlsForQuery = {
+  'recipients-with-no-tta': join(ssdiUrl, 'api', 'dashboards', 'qa', 'no-tta.sql'),
+  'recipients-with-ohs-standard-fei-goal': join(ssdiUrl, 'api', 'dashboards', 'qa', 'fei.sql'),
+  'recipients-with-class-scores-and-goals': join(ssdiUrl, 'api', 'dashboards', 'qa', 'class.sql'),
+  'qa-dashboard': join(ssdiUrl, 'api', 'dashboards', 'qa', 'dashboard.sql'),
+};
+
 const allowedTopicsForQuery = {
   'recipients-with-no-tta': [
+    'recipient',
+    'grantNumber',
+    'programType',
+    'stateCode',
     'region',
     'startDate',
     'endDate',
+    'dataSetSelection',
+  ],
+  'recipients-with-ohs-standard-fei-goal': [
+    'recipient',
     'grantNumber',
+    'programType',
     'stateCode',
-  ],
-  'recipients-with-ohs-standard-goal': [
     'region',
-    'startDate',
-    'endDate',
+    'group',
+    'createDate',
+    'reason',
+  ],
+  'recipients-with-class-scores-and-goals': [
+    'recipient',
     'grantNumber',
+    'programType',
     'stateCode',
+    'region',
+    'domainEmotionalSupport',
+    'domainClassroomOrganization',
+    'domainInstructionalSupport',
+    'createDate',
   ],
-  'qa-dashboard': [...QA_DASHBOARD_FILTER_CONFIG.map((filter) => filter.id), 'region'],
+  'qa-dashboard': [...QA_DASHBOARD_FILTER_CONFIG.map((filter) => filter),
+    'region',
+    'regionIds',
```

Comment:
why do we need both?

Reply:
I was waiting till Garrett updated the BE, I have removed it.